_tuning-your-cluster/availability-and-recovery/index.md

```diff
@@ -4,9 +4,8 @@ title: Availability and recovery
 nav_order: 20
 has_children: true
 has_toc: true
-permalink: /tuning-your-cluster/
 ---
 
 # Availability and recovery
 
-The following OpenSearch features help ensure reliability and consistent uptime so that your cluster can complete and scale based on your use case.
\ No newline at end of file
+The following OpenSearch features help ensure reliability and consistent uptime so that your cluster can complete and scale based on your use case.
```

Comment:
```suggestion
The following OpenSearch features help ensure reliability and consistent uptime so that your cluster can scale based on your use case.
```